Foundation/NSString.swift

```diff
@@ -971,28 +971,28 @@ extension NSString {
     }
     
     public var decomposedStringWithCanonicalMapping: String {
-        let string = CFStringCreateMutable(kCFAllocatorSystemDefault, 0)
+        let string = CFStringCreateMutable(kCFAllocatorSystemDefault, 0)!
```

Comment:
CFStringCreateMutable appears to have IUO return type imported. Maybe it hasn't been annotated for nullability.